### DISCUSSION TITLE
ScrollControlls pages with height greater than 100vh

### DISCUSSION THREAD

**Discussion by mpedraza10** [pmndrs/drei#1751] [Q&A]
I'm trying to use ScrollControls setting 4 pages for my 4 sections but one of them has a height greater than 100vh and makes the rest of the sections disappear. Is there a way around this issue by specifying that pages have a min-height of 100vh and still use the pages attribute?

**Comment by je0388** [comment]
ScrollControls assumes your height to be at max screen, i have searched for this a lot but still working by increasing pages. If you meet up with any solution do let me know. Also i want to know how can i have <Scroll>and <Scroll html > with different damping.
